framework/src/meshmodifiers/SideSetsAroundSubdomain.C

```diff
@@ -65,27 +61,13 @@ SideSetsAroundSubdomain::SideSetsAroundSubdomain(const InputParameters & paramet
 }
 
 void
-SideSetsAroundSubdomain::initialize()
-{
-  // Initialize the BlockRestrictable parent
-  initializeBlockRestrictable(_pars);
-}
-
-void
 SideSetsAroundSubdomain::modify()
 {
   // Reference the the libMesh::MeshBase
   MeshBase & mesh = _mesh_ptr->getMesh();
 
-  // Extract the 'first' block ID
-  SubdomainID block_id = *blockIDs().begin();
-
-  // Extract the SubdomainID
-  if (numBlocks() > 1)
-    mooseWarning("SideSetsAroundSubdomain only acts on a single subdomain, but multiple were "
-                 "provided: only the ",
-                 block_id,
-                 "' subdomain is being used.");
+  // Extract the block ID
+  SubdomainID block_id = (_mesh_ptr->getSubdomainIDs({getParam<SubdomainName>("block")}))[0];
```

Comment:
Oh I see. Well there is a better API that you can use that will clean this up, use the non-vector version:
https://github.com/idaholab/moose/blob/devel/framework/include/mesh/MooseMesh.h#L556

Then you can drop the braces and the brackets. After that I'll merge.